src/utils/Tmdb.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * TMDB API client configuration
+ * @type {{key: string, baseUrl: string, imageBaseUrl: string}}
+ */
+const apiConfig = {
+  key: '5265606bb69e99437c85eb04dc6f29b5',
+  baseUrl: 'https://api.themoviedb.org/3/',
+  imageBaseUrl: 'http://image.tmdb.org/t/p/',
+};
+
+/**
+ * An abstraction layer above TMDB API https://developers.themoviedb.org/3/getting-started/introduction
+ */
+class _Tmdb {
+  constructor() {
+    this.apiConfig = apiConfig;
+    this.imageConfig = undefined;
```

Comment:
Can you tell, what for do you need this variable?

Reply:
This is to cache response from `Tmdb.getImageryConfig()`. Refactored property name in c1c74fa